modules/permissions/common_permissions.tf

```diff
@@ -122,4 +122,20 @@ resource "aws_iam_policy" "custodian_get_parameters" {
   name        = "TDRCustodianGetParameters"
```

Comment:
Has `terraform fmt` been run on this file?

Reply:
No, I've run it now and on the other files.